src/Microsoft.AspNetCore.Mvc.Razor/RazorPage.cs

```diff
@@ -764,8 +763,7 @@ public bool IsSectionDefined(string name)
         /// In layout pages, renders the content of the section named <paramref name="name"/>.
         /// </summary>
         /// <param name="name">The name of the section to render.</param>
-        /// <returns>Returns <see cref="HtmlString.Empty"/> to allow the <see cref="Write(object)"/> call to
-        /// succeed.</returns>
+        /// <returns>An empty <see cref="IHtmlContent"/>.</returns>
         /// <remarks>The method writes to the <see cref="Output"/> and the value returned is a token
         /// value that allows the Write (produced due to @RenderSection(..)) to succeed. However the
         /// value does not represent the rendered content.</remarks>
```

Comment:
I think there's some value there though it's written in Nerd language. The valuable statement is that it doesn't actually return the text but rather it writes to the output. So there's probably a more succinct way to say it. But it's not too awful, and it's somewhat buried in the remarks...